tasks/aiida-rest.yml

```diff
@@ -16,13 +16,30 @@
     daemon_user: "{{ current_user }}"
     daemon_user_home: "{{ current_user_home }}"
     daemon_aiida_venv: "{{ aiida_venv |  regex_replace('\\$\\{HOME}', current_user_home) }}"
+  when: ansible_service_mgr == "systemd"
 
-- name: Start AiiDA REST API system service
+- name: Start AiiDA REST API service
   become: true
   become_user: "{{ root_user }}"
   systemd:
     name: aiida-rest@{{ aiida_profile_name }}
     enabled: "{{ aiida_rest_service_enabled }}"
     masked: false
     daemon-reload: true
-  when: not (cloud_platform is defined and cloud_platform == 'docker')
+  when: ansible_service_mgr == "systemd"
+
+- block:
+    - name: Check if the AiiDA REST API is running
```

Comment:
Perhaps better check that it is listening on the port we expect, i.e. [something like](https://stackoverflow.com/questions/28606876/ansible-check-if-service-is-listening-on-a-specific-port)

```
command: curl -sf localhost:5000/api/v4/server
```

Reply:
This only checks that the system is running and if we should start it again. My suggestion is that if we are going to check that things are actually running correctly I think we should do that in a separate PR. This also includes running a few AiiDA tests etc. Agreed?

Reply:
Fine.

Reply:
Awesome.

Reply:
Just 'unresolving' this to remember the fix when we do it.